met-web/src/components/engagement/form/EngagementWidgets/Documents/FileDrawer.tsx

```diff
@@ -140,10 +181,13 @@ const FileDrawer = () => {
                                     InputLabelProps={{
                                         shrink: false,
                                     }}
+                                    defaultValue={parentDocument?.title}
                                     fullWidth
                                     size="small"
                                 >
-                                    <MenuItem key={`folder-option-0`} value={0} sx={{ height: '2em' }}></MenuItem>
+                                    <MenuItem key={`folder-option-0`} value={0} sx={{ height: '2em' }}>
+                                        <i>none</i>
```

Comment:
just check if for italic there's a material UI prop in MenuItem, or maybe in the sx

Reply:
i added it into fontStyle: 'italic'